nuxeo-user-group-management/nuxeo-group-management.html

```diff
@@ -183,6 +183,10 @@
         text-align: center;
       }
 
+      .preserve-white-space {
+        white-space:pre;
```

Comment:
Missing space between property and value:
```suggestion
        white-space: pre;
```